src/Codeception/Module/REST.php

```diff
@@ -778,13 +803,16 @@ private function formatFilesArray(array $files)
                  * @issue https://github.com/Codeception/Codeception/issues/3298
                  */
             } else {
-                throw new ModuleException(__CLASS__, "Invalid value of key $name in files array");
+                throw new ModuleException(__CLASS__, sprintf('Invalid value of key %s in files array', $name));
             }
         }
 
         return $files;
     }
 
+    /**
+     * @return string|bool
+     */
     private function getFileType($file)
```

Comment:
This function returns string, because when mime_content_type returns false, this function returns `'application/octet-stream'` instead.